src/Uno.UWP/Storage/Streams/DataReader.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.Storage.Streams
+{
+	public partial class DataReader : IDataReader, IDisposable
+	{
+		private IBuffer _buffer = null;
+		private uint _position = 0;
+
+		public ByteOrder ByteOrder { get; set; }
+
+		public uint UnconsumedBufferLength
+		{
+			get => _buffer.Length - _position;
+		}
+		//public DataReader(global::Windows.Storage.Streams.IInputStream inputStream)
+		//{

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5527039)